### PR TITLE
Redux refresh part 1

### PR DESCRIPTION
**Pull request by xinranxiao**

### CODE REVIEW

File: client/build/.gitignore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```
Comment (tjk) [inline client/build/.gitignore]: Why did you remove this (and add the bundle)?

File: client/config/_base.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+var config = {
+  env: process.env.GO_ENV || 'development',
```
Comment (tjk) [inline client/config/_base.js]: What about just `ENV`?

File: client/src/containers/DevTools.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { createDevTools } from 'redux-devtools'
+import LogMonitor from 'redux-devtools-log-monitor'
+import DockMonitor from 'redux-devtools-dock-monitor'
+
+export default createDevTools(
+  <DockMonitor toggleVisibilityKey='ctrl-h' changePositionKey='ctrl-q'>
```
Comment (tjk) [inline client/src/containers/DevTools.js]: ` defaultIsVisible={false}`?

**Comment by tjk** [comment]
Minor comments but generally looks good to me. 

Please squash before merging (only to avoid committing the bundle blob in git).

File: client/config/_base.js (hunk 2 above)
Comment (xinranxiao) [inline client/config/_base.js]: sure
